src/services/notifier/utils/updaterUtils.ts

```diff
@@ -52,17 +55,21 @@ export const updateSubscriptionsBy = async (
 ): Promise<void> => {
   const [host, port] = providerUrl.split(/(?::)(\d*)$/, 2)
   const svcProvider = new NotifierSvcProvider({ host, port })
-  const subscriptionsDTO: any[] = await svcProvider.getSubscriptions(consumerAddress, subsHashesToUpdate)
+  try {
+    const subscriptionsDTO: any[] = await svcProvider.getSubscriptions(consumerAddress, subsHashesToUpdate)
 
-  await Promise.all(subscriptionsDTO.map(({
-    hash,
-    status,
-    paid,
-    notificationBalance
-  }) =>
-    SubscriptionModel.update(
-      { status, paid, notificationBalance },
-      { where: { hash } }
-    )
-  ))
+    await Promise.all(subscriptionsDTO.map(({
+      hash,
+      status,
+      paid,
+      notificationBalance
+    }) =>
+      SubscriptionModel.update(
+        { status, paid, notificationBalance },
+        { where: { hash } }
+      )
+    ))
+  } catch (error) {
+    logger.error('Unable to update subscriptions from notifier service provider', error)
+  }
```

Comment:
```suggestion
    await Promise.all(subscriptionsDTO.map(({
      hash,
      status,
      paid,
      notificationBalance
    }) =>
      SubscriptionModel.update(
        { status, paid, notificationBalance },
        { where: { hash } }
      ).catch((error) => {
        logger.error('Unable to update subscriptions from notifier service provider', error)
      })
    ))
```
and remove the try. Otherwise, it might not complete the processing the list.

Reply:
Thanks, added the chain catch to the update DB action.
We need to keep the try block to catch possible errors with communication with notifier service. In that case we won't update the DB and log the error.